.github/workflows/main.yml

```diff
@@ -65,23 +65,30 @@ jobs:
           auto-update-conda: true
           python-version: ${{ env.PYTHON_VER }}
           activate-environment: TESTconda
-          use-only-tar-bz2: true
+          use-only-tar-bz2: false
```

Comment:
This was using legacy conda packages and causing the conda env to fail to build. Should be true.

Reply:
Yes. Typo in my review/ explanation. We should not use tar-bz2